.github/CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 *                       @DataDog/agent-integrations
 /*/README.md            @DataDog/documentation
+/algorithmia/           support@algorithmia.io
```

Comment:
can you please add a github username or team here as well?

Reply:
Done. Added a Github username there. 👍